src/state.rs

```diff
@@ -40,6 +57,7 @@ where
         store,
         providers,
         supported_providers,
+        relay_client: RelayClient::new("https://relay.walletconnect.com".to_string()),
```

Comment:
Should the URL not be an env var to support multiple/custom envs?

/cc @arein

Reply:
There is a planned config refactor, saving this comment for that